core/proxy.c

```diff
@@ -397,6 +406,94 @@ oio_proxy_call_container_get_properties (CURL *h, struct oio_url_s *u,
 }
 
 GError *
+oio_proxy_call_content_get_properties (CURL *h, struct oio_url_s *u,
+					 GString **props_str)
+{
+	GString *http_url = _curl_content_url (u, "get_properties");
+	struct http_ctx_s out_ctx = {0};
+	if (props_str && *props_str)
+		out_ctx.body = *props_str;
+	else
+		out_ctx.body = g_string_sized_new(512);
+
+	GError *err = _proxy_call (h, "POST", http_url->str, NULL, &out_ctx);
+
+	g_string_free(http_url, TRUE);
+	if (props_str && !*props_str && !err)
+		*props_str = out_ctx.body;
+	else
+		g_string_free(out_ctx.body, TRUE);
+	return err;
+}
+
+GError *
+oio_proxy_call_container_set_properties (CURL *h, struct oio_url_s *u,
+				       const char* const *values) {
```

Comment:
The `{` is not properly indented. For a uniformity purpose, the opening parenthesis of functions must be on a line.